### ISSUE TITLE
Cannot connect to aria2 GUI when aria2 is connected to VPN

### ISSUE THREAD

**Issue by sallenko** [aria2/aria2#1404]
I have aria2 installed on NAS connected via USB to Asus router with Merlin firmware. Aria2 is installed through debian through hqt-guide.
I have a VPN client on my router and would like aria2 to download through that VPN (tun11). I noticed that I should be able to use the advance option interface and specify either ipadress or tunXX. When downloading with aria2 through putty and typing "--interface=tun11" it works.

For some reason when I type interface=tun11 in the aria2.conf file and try to connect to aria2 GUI, it will not connect. It says cannot connect to aria2 RPC server.

Is there a way to connect to aria2 GUI with interface=tun11 in aria2.conf file?
Do I need to adjust anything in the configuration.js file?

**Comment by sallenko** [comment]
Anyone?

**Comment by Plausible0Bug** [comment]
Isn't the answer obvious? you need to be able to access the same interface as aria2. Since it is now listening to a different interface.

**Comment by sallenko** [comment]
Yes, I read up on that later. Are you aware of any easy to change the interface of the aria2 gui rpc server?
